X-Account-Key: account5
X-UIDL: GmailId127e3030f5d1bffd
X-Mozilla-Status: 0000
X-Mozilla-Status2: 00000000
X-Mozilla-Keys:                                                                                 
Delivered-To: mlsubscriber.tech@csmining.org
Received: by 10.143.3.20 with SMTP id f20cs30612wfi;
        Fri, 9 Apr 2010 07:39:41 -0700 (PDT)
Received: by 10.223.98.83 with SMTP id p19mr170186fan.27.1270823980946;
        Fri, 09 Apr 2010 07:39:40 -0700 (PDT)
Return-Path: <bounce-debian-user=mlsubscriber.tech=csmining.org@lists.debian.org>
Received: from liszt.debian.org (liszt.debian.org [82.195.75.100])
        by mx.google.com with ESMTP id 25si3413106fxm.9.2010.04.09.07.39.40;
        Fri, 09 Apr 2010 07:39:40 -0700 (PDT)
Received-SPF: pass (google.com: manual fallback record for domain of bounce-debian-user=mlsubscriber.tech=csmining.org@lists.debian.org designates 82.195.75.100 as permitted sender) client-ip=82.195.75.100;
Authentication-Results: mx.google.com; spf=pass (google.com: manual fallback record for domain of bounce-debian-user=mlsubscriber.tech=csmining.org@lists.debian.org designates 82.195.75.100 as permitted sender) smtp.mail=bounce-debian-user=mlsubscriber.tech=csmining.org@lists.debian.org
Received: from localhost (localhost [127.0.0.1])
	by liszt.debian.org (Postfix) with QMQP
	id 85B742D1EA7; Fri,  9 Apr 2010 14:39:34 +0000 (UTC)
Old-Return-Path: <gldu-debian-user-2@m.gmane.org>
XChecker-Version: SpamAssassin 3.2.5 (2008-06-10) on liszt.debian.org
XLevel: 
XStatus: No, score=-9.7 required=4.0 tests=LDOSUBSCRIBER,LDO_WHITELIST,
	RCVD_NUMERIC_HELO autolearn=failed version=3.2.5
X-Original-To: lists-debian-user@liszt.debian.org
Delivered-To: lists-debian-user@liszt.debian.org
Received: from localhost (localhost [127.0.0.1])
	by liszt.debian.org (Postfix) with ESMTP id 6CBEB13A5A84
	for <lists-debian-user@liszt.debian.org>; Fri,  9 Apr 2010 14:39:28 +0000 (UTC)
X-Virus-Scanned: at lists.debian.org with policy bank en-ht
X-AmavisStatus: No, score=-4.933 tagged_above=-10000 required=5.3
	tests=[BAYES_00=-2, LDO_WHITELIST=-5, RCVD_NUMERIC_HELO=2.067]
	autolearn=no
Received: from liszt.debian.org ([127.0.0.1])
	by localhost (lists.debian.org [127.0.0.1]) (amavisd-new, port 2525)
	with ESMTP id xBYgkjFNOWYP for <lists-debian-user@liszt.debian.org>;
	Fri,  9 Apr 2010 14:39:21 +0000 (UTC)
X-policyd-weight: using cached result; rate:hard: -6.1
Received: from lo.gmane.org (lo.gmane.org [80.91.229.12])
	by liszt.debian.org (Postfix) with ESMTP id 4C6C013A5A71
	for <debian-user@lists.debian.org>; Fri,  9 Apr 2010 14:39:21 +0000 (UTC)
Received: from list by lo.gmane.org with local (Exim 4.69)
	(envelope-from <gldu-debian-user-2@m.gmane.org>)
	id 1O0FMa-0002SC-Kp
	for debian-user@lists.debian.org; Fri, 09 Apr 2010 16:39:16 +0200
Received: from 212.85.88.158 ([212.85.88.158])
        by main.gmane.org with esmtp (Gmexim 0.1 (Debian))
        id 1AlnuQ-0007hv-00
        for <debian-user@lists.debian.org>; Fri, 09 Apr 2010 16:39:16 +0200
Received: from johan.gronqvist by 212.85.88.158 with local (Gmexim 0.1 (Debian))
        id 1AlnuQ-0007hv-00
        for <debian-user@lists.debian.org>; Fri, 09 Apr 2010 16:39:16 +0200
X-Injected-Via-Gmane: http://gmane.org/
To: debian-user@lists.debian.org
From: =?ISO-8859-1?Q?Johan_Gr=F6nqvist?= <johan.gronqvist@csmining.org>
Subject: Re: Bug Report : Running Pymol for terminal produce ImportError:
 No  module named copy
Date: Fri, 09 Apr 2010 16:39:07 +0200
Lines: 26
Message-ID: <hpne6b$obp$1@dough.gmane.org>
References: <u2oafb39fcd1004090453of1b88637nf57ed0e6409ccdfa@mail.csmining.org>
Mime-Version: 1.0
Content-Type: text/plain; charset=ISO-8859-1; format=flowed
Content-Transfer-Encoding: 7bit
X-Complaints-To: usenet@dough.gmane.org
X-Gmane-NNTP-Posting-Host: 212.85.88.158
User-Agent: Mozilla-Thunderbird 2.0.0.22 (X11/20090707)
In-Reply-To: <u2oafb39fcd1004090453of1b88637nf57ed0e6409ccdfa@mail.csmining.org>
X-Rc-Virus: 2007-09-13_01
X-Rc-Spam: 2008-11-04_01
Resent-Message-ID: <ld2TANYG01M.A.4sE.mwzvLB@liszt>
Resent-From: debian-user@lists.debian.org
X-Mailing-List: <debian-user@lists.debian.org> archive/latest/573710
X-Loop: debian-user@lists.debian.org
List-Id: <debian-user.lists.debian.org>
List-Post: <mailto:debian-user@lists.debian.org>
List-Help: <mailto:debian-user-request@lists.debian.org?subject=help>
List-Subscribe: <mailto:debian-user-request@lists.debian.org?subject=subscribe>
List-Unsubscribe: <mailto:debian-user-request@lists.debian.org?subject=unsubscribe>
Precedence: list
Resent-Sender: debian-user-request@lists.debian.org
Resent-Date: Fri,  9 Apr 2010 14:39:34 +0000 (UTC)

Sushil Mishra skrev:
> When i try to open pymol by terminal thsi is producing fallowing error--
> 
> sushil@sushilnb:~$ pymol
> 'import site' failed; use -v for traceback
> Traceback (most recent call last):
>   File "/var/lib/python-support/python2.5/pymol//__init__.py", line 32, 
> in <module>
>     from copy import deepcopy
> ImportError: No module named copy
> sushil@sushilnb:~$
> 

I do not remember having had such a problem with pymol in debian. As I 
understand it, the command "from copy import deepcopy" is the first to 
run at all.

Given that the module copy is in the python standard library, I find it 
surprising that it is not found. Do you have any problems with other 
python programs?

Can you run the line
python -c "from copy import deepcopy"
without problems (errors)?

/ johan


-- 
To UNSUBSCRIBE, email to debian-user-REQUEST@lists.debian.org 
with a subject of "unsubscribe". Trouble? Contact listmaster@lists.debian.org
Archive: http://lists.debian.org/hpne6b$obp$1@dough.gmane.org


